lib/ts/recipe/accountlinking/recipe.ts

```diff
@@ -202,10 +217,20 @@ export default class Recipe extends RecipeModule {
             return user;
         }
 
+        const accountInfo: AccountInfoInput = {
+            ...user.loginMethods[0],
+            webauthn: undefined,
+        };
+        // any credentialId will identify the same user
+        if (user.loginMethods[0]?.webauthn?.credentialIds.length) {
+            accountInfo.webauthn = {
+                credentialId: user.loginMethods[0].webauthn.credentialIds[0],
+            };
+        }
```

Comment:
Here it may also not make sense to search using the `credentialId`, since it'll only match the current user which is already a part of the results because the email address is already present.

Reply:
Done